barustenberg/src/plonk/composer/composer_base.rs

```diff
@@ -20,6 +20,8 @@ pub(crate) const FIRST_VARIABLE_IN_CLASS: u32 = u32::MAX - 2;
 pub(crate) const NUM_RESERVED_GATES: usize = 4;
 
 #[derive(Debug, PartialEq, Eq, Clone, Copy)]
+#[allow(clippy::enum_clike_unportable_variant)]
+// Note that this will fail to compile on 32-bit systems
```

Comment:
can we figure out how to just block it from compiling on a 32-bit system? does this flag do that?

Reply:
This one will stop the compile errors, but tbh i don't think we need to worry about 32bit systems.

Reply:
Yeah we'll add a readme note eventually... but wasm, no?